setup.py

```diff
@@ -41,31 +41,23 @@ def run(self):
     namespace_packages=['chevah'],
     packages=find_packages('.'),
     package_data={
-        'chevah.github_hooks_server': ['static/*', 'author-aliases.txt'],
+        'chevah.github_hooks_server': ['author-aliases.txt'],
         },
-    scripts=['scripts/start-chevah-github-hooks.py'],
     install_requires=[
-        'klein==17.2',
         'toml',
-        # We keep an older version of python is use.
-        'Twisted==15.5.0.chevah1',
-        'github3-py==1.0.0.gitc82e90e',
-        # We don't depended directly on them but github3.py
-        # dependencies are bad.
-        'urllib3',
-        'chardet',
-        'certifi',
-        'idna',
+        'github3.py',
+        'azure-functions',
+        'azure-functions-worker',
         ],
-    extras_require = {
+    extras_require={
```

Comment:
don't worry too much about formating... we will move this to `setup.cfg` or `pyproject.toml` 

we might want to experiment with https://python-poetry.org/

Reply:
I added a tentative `pyproject.toml`. You can use it with `poetry shell`, then run what you need.

Reply:
have you pushed it ? I don't see any pyproject.toml  ... and I think that once you use pyproject.toml you can remove the setup.py  ... or have just `setup()` inside setup.py to have `pip install =e .`

Now that I am looking at poetry I see that it is sponsored by vercel and that vercel has python serverless functions

https://vercel.com/docs/concepts/functions/supported-languages#supported-languages

so maybe we can deploy this to vercel instead of azure :p

Reply:
No, I commented before pushing; sorry. I was experimenting with the venv also; I have pushed now.

I think I would also prefer not to use this library with tons of insecure dependencies. When pushing I got:
```
remote: GitHub found 5 vulnerabilities on chevah/github-hooks-server's default branch (2 critical, 1 high, 2 moderate). To find out more, visit:
remote:      https://github.com/chevah/github-hooks-server/security/dependabot
remote: 
```

But the pros of serverless is we can easily change to [cheaper providers](https://www.serverless.com/framework/docs/providers). Looking at the [Azure intro](https://www.serverless.com/framework/docs/providers/azure/guide/intro/) from there makes me think we don't need the `serverless-azure-functions` plugin.

I like whatever is developer-friendly and not too costly to adapt (to avoid vendor lock-in).

Reply:
we tried this "technology" 

for huge projects it might make sense to make it easy to switch between providers ... but for us this is not a big deal.

I went with Azure Functions as it had support for python serverless.

I now see that vercel also has support for Python

but it looks like the future is nodejs for serverless ...so in the end we might just embrace JS for all web related things